src/MessageQueue/MessageQueue.php

```diff
@@ -109,6 +107,12 @@ private function doFlush(bool $async)
             foreach ($messagesToPush as $key => $message) {
                 $messageParameters = [
                     'Id'           => $key, // Identifier of the message in the batch
+                    'MessageAttributes' => [
+                        'name' => [
```

Comment:
Should we name it "Name" for consistency with the header which is "X-Aws-Sqsd-Attr-*" so it would look X-Aws-Sqsd-Attr-Name :p.